File: Leetcode/2068/2068.cpp
class Solution {
public:
    bool checkAlmostEquivalent(string word1, string word2) {
        constexpr int MAXN = 26;
        int stats1[MAXN]{};
        for (auto&i : word1) ++stats1[i-'a'];
        int stats2[MAXN]{};
        for (auto&i : word2) ++stats2[i-'a'];
        for (int i = 0; i < MAXN; ++i) {
            if (abs(stats1[i]-stats2[i]) > 3) return false;
        }
        return true;
    }
};

File: Leetcode/2068/2068.go
func abs(x int) int {
	if x < 0 {
		return -x
	}
	return x
}

func checkAlmostEquivalent(word1 string, word2 string) bool {
	const N = 26
	freq1 := make([]int, N)
	freq2 := make([]int, N)
	for i := 0; i < len(word1); i++ {
		freq1[word1[i]-'a']++
	}
	for i := 0; i < len(word2); i++ {
		freq2[word2[i]-'a']++
	}
	const THRESHOLD = 3
	for i := 0; i < N; i++ {
		if abs(freq1[i] - freq2[i]) > THRESHOLD {
			return false
		}
	}
	return true
}